README.md

```diff
@@ -0,0 +1,37 @@
+# terraform-aws-lb-listener-certificate
+
+Provides a Load Balancer Listener Certificate resource.
+
+This resource adds an ACM certificate to the load balancer for the specified
+listeners. It does *not* replace the default certificate on the ALB.
```

Comment:
Perhaps. But yet, we have never made a NLB, nor have I tested the code's ability to do so. I suppose we should just say "LB" because that's part of the module's name, but I will note that many of the Terraform modules uses LB and ALB interchangeably.

Reply:
I don't think we need to test this with an NLB. The Terraform resource [aws_lb](https://www.terraform.io/docs/providers/aws/r/lb.html) is used to create both Network Load Balancers and Application Load Balancers. Both use the same listeners resource [aws_lb_listener](https://www.terraform.io/docs/providers/aws/r/lb_listener.html) which is what we actually attach the ACM certs too. There is no reason to mention ALB. Just say Load Balancer.